docs/admonitions.rst

```diff
@@ -2,219 +2,309 @@
 Admonitions
 ===========
 
-This theme uses the :dutree:`admonition` directives for Sphinx admonitions.
+This theme uses the `Admonition Directive`_ for Sphinx admonitions.
+
+.. _predefined_admonitions:
 
 rST and Sphinx
 **************
 
 Most of the admonitions that the mkdocs-material theme supports were "borrowed" from
 admonitions defined in the reStructuredText specifications. You may recognize them from
-usage in other sphinx-based themes. They are:
+usage in other sphinx-based themes.
 
-.. rst-example:: ``note``, ``todo``, ``seealso``
+.. rst-example:: ``note``, ``seealso``, ``todo``
+   :name: note-style
 
    .. seealso::
       This admonition is specific to Sphinx directives and not defined in the reStructuredText
-      specifications as you can `seealso`. The `todo` admonition is also defined by Sphinx.
+      specifications as you can `seealso`. The `todo` admonition is also defined as an extension
+      that is distributed with Sphinx.
 
       The :dutree:`note` admonition *is* defined by the reStructuredText specifications.
 
 .. rst-example:: ``tip``, ``hint``, ``important``
+   :name: important-style
 
    .. important::
       It is :dutree:`important` to correctly use admonitions.
 
 .. rst-example:: ``attention``, ``caution``, ``warning``
+   :name: attention-style caution-style warning-style
 
    .. warning::
       This is a :dutree:`warning`.
 
 .. rst-example:: ``danger``, ``error``
+   :name: error-style
 
    .. error::
       You have made a grave :dutree:`error`.
 
+.. _inherited_admonitions:
+
 Admonitions from mkdocs-material
 ********************************
 
-Some additional admonitions are supported via the source code from the mkdocs-material theme.
-These admonitions can still be used, but the syntax is a little different because it relies
-on the generic :dutree:`admonition` defined in the reStructuredText specifications.
-
-To use the following admonitions' styles from the mkdocs-material theme, the rST syntax is
-shown to demonstrate using the :rst:`:class:` option of generic admonitions.
+Some additional admonitions are supported via the CSS styles inherited from the mkdocs-material
```

Comment:
Perhaps we should make these optional, in case they conflict with another extension or user-defined directive?

Reply:
Good idea. I was worried about this also because the terse names used (`bug`). I also considered using the `si` prefix for only the directive name.

Reply:
I have 
- renamed the inherited admonitions' directives with a `si-` prefix.
- added a config value that allows disabling generating these convenience directives (defaults to True)